feelpp/feel/CMakeLists.txt

```diff
@@ -171,7 +171,7 @@ if ( GMSH_FOUND )
   if ( FEELPP_HAS_MED )
     target_compile_definitions( feelpp PUBLIC FEELPP_HAS_MED FEELPP_HAS_GMSH_HAS_MED )
   endif()
-  target_include_directories( feelpp PUBLIC ${GMSH_INCLUDE_DIR} )
+  target_include_directories( feelpp BEFORE PUBLIC ${GMSH_INCLUDE_DIR} )
```

Comment:
Ok, but this should then be discussed quickly, as it can break compilation, so that I need to provide help for the compilation to people who use this toolbox...

Reply:
Could you extract it in a small pr then?

Reply:
you can leave it here and it will get merged with the small pr which will be addressed probably before

Reply:
Yeah sure, that's also what I do. The PR is here : #1289.